src/js/utils.js

```diff
@@ -178,3 +178,15 @@ export function checkIterate(arr, func) {
     })
 }
 
+export function formatNumericalValue(number, formatting, method) {
+    let fn = null;
```

Comment:
why not define this as `let fn = d3format(formatting.integer);` then you can move the else block to another `if` and have a nice default.

Reply:
I changed it as 
    ```
let fn = d3format(formatting.percentage);
    if (method === 'absolute_value') {
        fn = d3format(formatting.integer);
    } else if (method === 'decimal') {
        fn = d3format(formatting.decimal);
    }
```
I used percentage as default instead of integer because most of the values are percentages

Reply:
👍